app/src/main/java/pl/droidevs/books/repository/BookRepository.java

```diff
@@ -0,0 +1,32 @@
+package pl.droidevs.books.repository;
+
+import android.arch.lifecycle.LiveData;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import pl.droidevs.books.app.BookDataBase;
+import pl.droidevs.books.entity.Book;
+
+public class BookRepository {
+
+    @Inject
+    BookDataBase bookDataBase;
```

Comment:
Jeśli ustawicz @Inject na paremetrze konstruktora to pole będzie mogło być prywatne.